test/integration/spec/room.js

```diff
@@ -241,8 +246,12 @@ describe('Room', function() {
   });
 
   describe('"disconnected" event', () => {
-    it.skip('is raised whenever the LocalParticipant is disconnected via the REST API', () => {
-      // TODO(mroberts): POST to the REST API to disconnect the LocalParticipant from the Room.
+    it('is raised whenever the LocalParticipant is disconnected via the REST API', async () => {
```

Comment:
> So I'm assuming this test will run for both P2P and Group Rooms?

Yes, no reason for it not to, AFAICT—it's applicable in both scenarios and should behave the same.

> I assumed you would add this test to `test/integration/spec/rest.js`?

Since we had planned on this test a while back, per the `TODO`, I just filled it in. Re: `test/integration/spec/rest.js`, actually, I'm not sure we need a test module specifically for "REST" tests; rather, I think REST tests could be included within our existing modules.